### PR TITLE
[spec] Add spec for collecting ansible_host

### PR DESCRIPTION
**Pull request by SteveHNH**
The client will begin collecting ansible_host and putting it into the
root of the archive. This spec will let us dig it out of puptoo for
system_profile

RHCLOUD-13536

Signed-off-by: Stephen Adams <tsadams@gmail.com>

### CODE REVIEW

**Comment by bfahr** [comment]
@SteveHNH I submitted #3008 for @gravitypriest but I'll close it and use this one instead.  It also needs to go through spec approval.

**Review by bfahr — APPROVED** [review]
Approved pending spec approval

File: insights/specs/insights_archive.py [hunk 1]
```diff
@@ -13,6 +13,7 @@ class InsightsArchiveSpecs(Specs):
     abrt_status_bare = simple_file("insights_commands/abrt_status_--bare_True")
     all_installed_rpms = glob_file("insights_commands/rpm_-qa*")
     alternatives_display_python = simple_file("insights_commands/alternatives_--display_python")
+    ansible_host = simple_file("ansible_host")
```
Comment (bfahr) [inline insights/specs/insights_archive.py:16]: This change is not required, it will be covered by the `Core3Specs`.

**Review by bfahr — CHANGES_REQUESTED** [review]

**Comment by SteveHNH** [comment]
@bfahr all fixed up

File: insights/specs/core3_archive.py [hunk 2]
```diff
@@ -14,5 +14,4 @@
 
 class Core3Specs(Specs):
     branch_info = simple_file("/branch_info", kind=RawFileProvider)
-    display_name = simple_file("display_name")
```
Comment (bfahr) [inline insights/specs/core3_archive.py:17]: @SteveHNH leave this here, and add `ansible_host` here as well to support core processing of core-collection archives.

File: insights/specs/default.py [hunk 3]
```diff
@@ -133,6 +133,7 @@ class DefaultSpecs(Specs):
     abrt_status_bare = simple_command("/usr/bin/abrt status --bare=True")
     alternatives_display_python = simple_command("/usr/sbin/alternatives --display python")
     amq_broker = glob_file("/var/opt/amq-broker/*/etc/broker.xml")
+    ansible_host = simple_file("ansible_host")
```
Comment (bfahr) [inline insights/specs/default.py:136]: Remove this spec and add it to `insights_archive.py` to support reading non-core-collection archives.

File: insights/specs/default.py [hunk 4]
```diff
@@ -170,6 +171,7 @@ def is_azure(broker):
     brctl_show = simple_command("/usr/sbin/brctl show")
     candlepin_log = simple_file("/var/log/candlepin/candlepin.log")
     cgroups = simple_file("/proc/cgroups")
+    display_name = simple_file("display_name")
```
Comment (bfahr) [inline insights/specs/default.py:174]: Remove this spec, its already in `insights_archive.py`.

**Review by bfahr — CHANGES_REQUESTED** [review]

**Review by bfahr — APPROVED** [review]

**Pull request merged**